addons/finger/functions/fnc_incomingFinger.sqf

```diff
@@ -0,0 +1,35 @@
+/*
+ * Author: TheDrill, PabstMirror
+ * Recieve an finger event, adds to the array (or updates if already present) and starts PFEH if not already running
+ *
+ * Arguments:
+ * 0: Source Unit (can be self) <OBJECT>
+ * 1: Position being pointed at (from positionCameraToWorld) <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [bob, [1,2,3]] call ace_finger_fnc_incomingFinger;
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+PARAMS_2(_sourceUnit,_fingerPosPrecise);
+
+private ["_data", "_fingerPos"];
+
+//add some random float to location if it's not our own finger:
+_fingerPos = if (_sourceUnit == ACE_player) then {
+    _fingerPosPrecise
+} else {
+    _fingerPosPrecise vectorAdd [random (2*FP_RANDOMIZATION_X) - FP_RANDOMIZATION_X, random (2*FP_RANDOMIZATION_X) - FP_RANDOMIZATION_X, random (2*FP_RANDOMIZATION_Y) - FP_RANDOMIZATION_Y]
```

Comment:
I'm not sure adding noise is really needed. Pointing IRL is really accurate, certainly inmensely more than the 20º error we are adding (+-350m error at 1000m distance is way way too much). I think this will be even more confusing if we allow pointing at closer objects.